report.tex

```diff
@@ -15,6 +24,21 @@
 \bibliographystyle{plain}	% Literaturverzeichnisformat = standard
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%OTHER OPTIONS
+\setlength{\parindent}{0px} %dieser Befehl verhindert das Einrücken eines Absatzes bei einer leeren Codezeile. 
+
+%SHORTCUTS
+\newcommand{\R}{\mathbb{R}} %reelle Zahlen
+\newcommand{\N}{\mathbb{N}} %natürliche Zahlen
+\newcommand{\Z}{\mathbb{Z}} %ganze Zahlen
+\newcommand{\C}{\mathbb{C}} %komplexe Zahlen
+
+\newcommand{\stackeq}[1]{\mathrel{\stackrel{\makebox[0pt]{\mbox{\normalfont\tiny #1}}}{=}}} %das ist ein Gleichheitszeichen mit Text darüber, Beispiel: $a\stackeq{Def} b$
+\newcommand{\stacksymbol}[2]{\mathrel{\stackrel{\makebox[0pt]{\mbox{\normalfont\tiny #1}}}{#2}}} %das ist ein beliebiges Zeichen mit Text darüber, z. B. $a\stackrel{Def}{\Rightarrow} b$
+\newcommand\tab[1][1cm]{\hspace*{#1}} %praktischer Tabulator
```

Comment:
What does this mean?

Reply:
Do you mean the \tab-Command? 

This is just a selfmade tabulator for formatting source code etc. in LaTeX.